x-pack/legacy/plugins/integrations_manager/server/packages/remove.ts

```diff
@@ -21,11 +23,22 @@ export async function removeInstallation(options: {
   await savedObjectsClient.delete(SAVED_OBJECT_TYPE, pkgkey);
 
   // Delete the installed assets
-  const deletePromises = installedObjects.map(async ({ id, type }) =>
-    savedObjectsClient.delete(type, id)
-  );
+  const deletePromises = installedObjects.map(async ({ id, type }) => {
+    if (assetUsesObjects(type as AssetType)) {
+      savedObjectsClient.delete(type, id);
+    } else if (type === AssetType.ingestPipeline) {
+      deletePipeline(callCluster, id);
+    }
+  });
   await Promise.all(deletePromises);
 
   // successful delete's in SO client return {}. return something more useful
   return installedObjects;
 }
+
+async function deletePipeline(callCluster: CallESAsCurrentUser, id: string): Promise<any> {
```

Comment:
I think this is
```suggestion
async function deletePipeline(callCluster: CallESAsCurrentUser, id: string): Promise<void> {
```

because never return a value

Reply:
Good catch, thank you!